app/routes/polls/components/PollEditor.js

```diff
@@ -0,0 +1,176 @@
+// @flow
+
+import React, { Component } from 'react';
+import { Content } from 'app/components/Content';
+import NavigationTab, { NavigationLink } from 'app/components/NavigationTab';
+import Button from 'app/components/Button';
+import Icon from 'app/components/Icon';
+import { Link } from 'react-router';
+import {
+  TextInput,
+  SelectInput,
+  TextArea,
+  legoForm,
+  CheckBox
+} from 'app/components/Form';
+import { Form, Field, FieldArray } from 'redux-form';
+import { ConfirmModalWithParent } from 'app/components/Modal/ConfirmModal';
+import styles from './PollEditor.css'
+
+const renderOptions = ({ fields, meta: { touched, error } }) => {
+  return [
+    <ul className={styles.options} key="options">
+      {fields.map((option, i) => (
+        <li className={styles.optionField} key={i}>
+        <Field
+          name={`${option}.name`}
+          label={`Valg nr. ${i+1}`}
+          placeholder={`Valg ${i+1}`}
+          component={TextInput.Field}
+          required
+        />
+        <ConfirmModalWithParent
+          title="Slett valg"
+          message="Er du sikker på at du vil slette dette valget?"
+          onConfirm={() => Promise.resolve(fields.remove(i))}
+          closeOnConfirm
+          className={styles.deleteOption}
+        >
+          <Icon name="trash" className={styles.deleteOption}/>
+        </ConfirmModalWithParent>
+        </li>
+      ))}
+    </ul>,
+
+    <Link
+      key="addNew"
+      onClick={() => {
+        fields.push({});
+      }}
+      className={styles.addOption}
+    >
+      <Icon name="add-circle" size={25}/> Valg
+    </Link>
+  ];
+};
+
+type Props = {
+  pristine: Boolean,
+  submitting: Boolean,
+  actionGrant: Array<String>,
```

Comment:
This type seems to be unused in this file

Reply:
Yep, removed it from props also.